playbooks/operationalizing/saml_sso.adoc

```diff
@@ -0,0 +1,394 @@
+# SAML Authentication for OpenShift
```

Comment:
Upon beginning inspection of the doc, it looks like most of this is in markdown format, not asciidoc. We need this to be asciidoc formatted in order to publish to the site.

Check out the style guide here: https://github.com/rhtconsulting/openshift-playbooks/blob/master/style_guide.adoc